Global.h

```diff
@@ -698,3 +698,16 @@ namespace vstd
 }
 using vstd::operator-=;
 using vstd::make_unique;
+
+#if defined VCMI_ANDROID && defined NO_STD_TOSTRING

```

Comment:
It looks like std::to_string works when I add both -std=gnu++11 and _GLIBCXX_USE_C99 (which is internal define, so that might be a bad idea...)